packages/types/src/index.ts

```diff
@@ -1,10 +1,10 @@
+export * from './block-kit/block-elements';
+export * from './block-kit/blocks';
+export * from './block-kit/composition-objects';
+export * from './block-kit/extensions';
 export * from './calls';
 export * from './dialog';
 export * from './events';
-export * from './message-metadata';
 export * from './message-attachments';
+export * from './message-metadata';
 export * from './views';
```

Comment:
👁‍🗨 The `*` export worries me sometimes but I believe we're safe against name collisions in the build step.